shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-opengauss/src/test/java/org/apache/shardingsphere/proxy/frontend/opengauss/OpenGaussFrontendEngineTest.java

```diff
@@ -56,9 +55,6 @@ public void setup() {
     private void prepareMock() {
         Field field = OpenGaussFrontendEngine.class.getDeclaredField("postgreSQLFrontendEngine");
         field.setAccessible(true);
-        Field modifiers = Field.class.getDeclaredField("modifiers");
```

Comment:
Same problem with previous one.

Reply:
Same reason as above, due to the merge conflict.